ivandocs/validation.md

```diff
@@ -0,0 +1,64 @@
+| 1a 	| Confirm location updates are configured in iOS. Also ensure that you have the appropriate plist keys setup in iOS to request user permission to track location: https://placesdocs.com/places-services-by-adobe-documentation/configure-places-in-the-sdk/places-monitor-extension/using-the-places-monitor-extension#enable-location-updates-in-background 	| Confirmed 	|
+| 2 	| Confirm which monitoring mode is set for iOS. Note continuous mode allows for greater accuracy and persistence, but also more heavily drains battery life: https://placesdocs.com/places-services-by-adobe-documentation/configure-places-in-the-sdk/places-monitor-extension/places-monitor-api-reference#monitoring-mode-ios-only 	| Significant Changes or Continuous 	|
+| 3 	| If using more than a single Library of POIs, confirm that the appropriate libraries have been selected in the Places extension for Launch 	| Confirmed |
+| 4 	| Confirm that the latest versions of Mobile Core and Places / Places Monitor extensions have been bundled with the app via Gradle or CocoaPods	| Confirmed - for more information on recent updates see: https://placesdocs.com/places-services-by-adobe-documentation/release-notes 	|
+| 5 	| Confirm that the correct environments are configured for testing. Launch environment ID should match your Launch development environment 	|  Confirmed |
+| 6 	| Create GPX files for each POI that you want to test. GPX files can be used in local development environment to simulate a location entry. See links below for information on creating and using GPX files. <br>https://stackoverflow.com/questions/17292783/gpx-files-for-ios-simulator<br>https://mapstogpx.com/mobiledev.php<br>https://blackpixel.com/writing/2013/05/simulating-locations-with-xcode.html<br>https://qacumtester.wordpress.com/2014/02/27/location-testing-in-mobile-apps/ 	| GPX files created and loaded into app project 	|
+| 7 	| Without doing anything else, you should be able to launch the application from Android Studio or XCode and see the appropriate alert to request access for tracking location. Grant "Always Allow" permission.<br>*Recommend using a real device connected to your computer over using device simulator.* 	| Location request prompt should display on application loaded through IDE 	|
+| 8 	| Once Location permission has been accepted you should see in the application console messages that indicate ```"ACPExtensionEventName" : "requestgetnearbyplaces"```, has been called. Switching between different locations in XCode or Android studio should produce entry events for specific POIs  	| "ACPExtensionEventName" : "requestgetnearbyplaces" should be displayed as you simulate different locations.	|
```

Comment:
POINT 8

**Description**
Once Location permission has been accepted. The Places SDK will to retrieve the current location of the device and the Places Monitor extension will start monitoring the 20 closest POIs from the current location


**iOS** 
```[AdobeExperienceSDK DEBUG <com.adobe.placesMonitor>]: Authorization status changed: Always
[AdobeExperienceSDK DEBUG <Places>]: Requesting 20 nearby POIs for device location (<lat>, <longitude>)
[AdobeExperienceSDK DEBUG <Places>]: Response from Places Query Service contained <n> nearby POIs
[AdobeExperienceSDK DEBUG <com.adobe.placesMonitor>]: Received a new list of POIs from Places: (
"<ACPPlacePoi: 0x600002b75a40> Name: <poi name>; ID:<poi id>; Center: (<lat>, <long>); Radius: <radius>”,
..
..)
```


**Android**

```PlacesMonitor - All location settings are satisfied to monitor location
PlacesMonitor - PlacesMonitorInternal : New location obtained: <latitude> <longitude> Attempting to get the near by pois
PlacesExtension - Dispatching nearby places event with n POIs
PlacesMonitor - Attempting to Monitor POI with id <poi id> name <poi name> latitude <lat> longitude <longitude>
PlacesMonitor - Attempting to Monitor POI with id <poi id> name <poi name> latitude <lat> longitude <longitude>
PlacesMonitor - Attempting to Monitor POI with id <poi id> name <poi name> latitude <lat> longitude <longitude>
….
PlacesMonitor - Successfully added n fences for monitoring

```